### DISCUSSION TITLE
Where can we see moved data in backupstoragelocation ?

### DISCUSSION THREAD

**Discussion by navilg** [vmware-tanzu/velero#7120] [Community support Q&A]
Hello,

I was testing data mover with csi-snapshot with velero 1.12. Backup was successful. I was trying to find out where in backupstorage location do the data is stored ? But I am not able to find that out.

Could you please help me with the path/folder structure in backup storage location where data from csi-snapshots are moved and stored ?

@Lyndon-Li Would you please be able to help me with the query ?

Below is the screenshot of my minio bucket used for testing:
![image](https://github.com/vmware-tanzu/velero/assets/37469234/2e85d36b-f8fe-480a-9d78-02dabba0571e)

![image](https://github.com/vmware-tanzu/velero/assets/37469234/d7e6096e-f751-4928-8b2d-092d266d7c1e)

**Comment by navilg** [comment]
backup-3 is the backup created using velero data-mover.

**Comment by Lyndon-Li** [comment]
Volume backup data is not stored as the original formats, data are deduplicated and encrypted, so you cannot identify the data from the backup storage.
Normally, you can review the backed up data by restoring the backup to a cluster. Otherwise, if you want to see the data without restore anyway, you can use Kopia's mount functionality https://kopia.io/docs/mounting/

**Comment by hazeelowod35** [comment]
"The backed-up volume data undergoes deduplication and encryption, altering the original formats. Consequently, the data stored in backup storage is not identifiable. Can I obtain backup data for my this site [http://www.appvalleyz.com/riding-extreme-3d-mod-apk/](https://appvalleyz.com/riding-extreme-3d-mod-apk/)?"
